### DISCUSSION TITLE
Screenshot of UI in readme

### DISCUSSION THREAD

**Discussion by koma2412** [itsfourthidiot/ira#60] [General]
It'll be great if we can see UI screenshot in readme. Because people like me want to go through the look and features(to maybe provide some suggestions) but dont have softwares installed to host the code.

**Comment by itsfourthidiot** [comment]
We will add UI screenshots in a few days. Till then, you can go have a look at our application through demo videos:
- Sprint 1 demo [[Link](https://www.youtube.com/watch?v=8qk0wVWVb_s)]
- Sprint 2 backend demo [[Link](https://www.youtube.com/watch?v=uJ0GS2_R4Yw)]
- Sprint 2 frontend demo [[Link](https://youtu.be/cypa1Fu46HQ)]

**Comment by itsfourthidiot** [comment]
Hey, here you can see the UI screenshots of our application:
https://github.com/fourth-idiot/ira/wiki/Sprint-3-UI-Screenshots
